node/src/main.rs

```diff
@@ -235,6 +237,20 @@ async fn main() {
                      (e.g. 'ethereum/mainnet').",
                 ),
         )
+        .arg(
+            Arg::with_name("arweave-http")
+                .default_value("https://arweave.net/")
+                .long("arweave-http")
+                .value_name("URL")
+                .help("HTTP endpoint of an Arweave gateway"),
+        )
+        .arg(
+            Arg::with_name("3box-profile-http")
```

Comment:
Internally they refer to this as the "profile server" https://github.com/3box/3box-js/blob/510137adbdf3ef4e240d9a7789946e967a19ff30/src/config.js#L31

Reply:
I'm proposing a more general term, because the `ipfs.3box.io` API provides more than just profiles, it also provides spaces and the user's 3box config.

Reply:
@leoyvens From my perspective, this is the last outstanding item.

Reply:
What about `--3box-ipfs`?

Reply:
I think this is the only API they have. I'm not even sure why it's got an `ipfs` subdoain on it tbh.

Reply:
Ah, ok it's `--3box-api` now.